src/main/java/Main.java

```diff
@@ -31,6 +32,7 @@ public static void main(String[] args) {
         EnrolmentPresenter enrolmentPresenter = new EnrolmentPresenter(databaseDriver);
         RunCoursePresenter coursePresenter = new RunCoursePresenter(databaseDriver);
         EvaluatorPresenter evaluatorPresenter = new EvaluatorPresenter(databaseDriver);
+        CreateCoursePresenter createCoursePresenter = new CreateCoursePresenter(databaseDriver);
```

Comment:
I think this needs to be refactored so that it's instead set up in the screen

Reply:
You mean line 32,33,34and 35 or just line35?